modules/@apostrophecms/schema/lib/joinr.js

```diff
@@ -68,6 +68,14 @@ const joinr = module.exports = {
     getter,
     idMapper
   ) {
+    // This method never alters the items array itself, it alters
```

Comment:
Note that reducing to unique elements in this way does not make new objects nor does it test actual object contents. This is intentional. We just want to eliminate duplicates *by reference*.